casper/src/main/scala/coop/rchain/casper/util/comm/ListenAtName.scala

```diff
@@ -25,21 +25,21 @@ object ListenAtName {
 
   private def buildPar[G[_]: BuildPar](name: G[Name]) = implicitly[BuildPar[G]].build(name)
 
-  implicit def buildParF[F[_]: MonadError[?[_], Throwable]] = new BuildPar[λ[A => F[Id[A]]]] {
+  implicit def buildParF[F[_]: MonadError[*[_], Throwable]] = new BuildPar[λ[A => F[Id[A]]]] {
```

Comment:
What is the difference between `?` and `*`? 

I thought that it doesn't care which charactor it used.

Reply:
I think Scala folks just want to make thing more interesting. :smiley:

I think the main reason is to support partial application of type level functions.
```scala
F[_, B](A)  =>  F[A, B]
F[A, _](B)  =>  F[A, B]
```